### ISSUE TITLE
Installer unknown locale error

### ISSUE THREAD

**Issue by End45** [SkyTemple/skytemple#674]
Originally reported at https://discord.com/channels/710190644152369162/1183080667110703176

Seems like the SkyTemple 1.6.0 installer fails with certain locales.
![Locale error](https://github.com/SkyTemple/skytemple/assets/71291438/0e3fc550-5f45-4369-b266-85854cf4beb8)

**Comment by theCapypara** [comment]
Duplicate of #672

**Comment by End45** [comment]
> Duplicate of #672

The error message is not the same, are we sure it's the same issue?

**Comment by theCapypara** [comment]
ah, you are right. The error is not caught here. However I'm not really sure how to fix this, other than also just catching and ignoring the error. This seems to be an upstream Python bug, en_ID is a pretty new locale.

**Comment by End45** [comment]
Reopening this as there's been another report (https://discord.com/channels/710190644152369162/1246112469055967254)
![Locale error](https://github.com/SkyTemple/skytemple/assets/71291438/0314b118-eda7-41b6-b671-c46b4d6ece55)

**Comment by theCapypara** [comment]
This is an dependency, we can't fix this.

**Comment by End45** [comment]
Not even by catching and ignoring the error as you said above?

**Comment by theCapypara** [comment]
I don't think so at that point. the call is made by cairosvg to the builtin locale module, which just fails with this error. We could hot-patch locale. Maybe.
https://docs.python.org/3/library/locale.html

**Comment by theCapypara** [comment]
I still can't find anybody else with this issue, and I don't really understand how the locale module is supposed to work under Windows. I don't really know where the actual issue is.

**Comment by End45** [comment]
A workaround has been found: https://discord.com/channels/710190644152369162/1259252798449651864
Maybe that can help clarify why the issue happens?

**Comment by theCapypara** [comment]
The author of that thread changed their locale to a supported one.
Sure that is a workaround, but it doesn't really help, some Windows people would need to investigate this with Python and potentially raise an upstream bug with them or PyInstaller, depending on where the bug is.

**Comment by End45** [comment]
I see. Oh well, at least now we have proper instructions on what to do to avoid the problem.
